components/brave_ads/core/internal/ads/serving/targeting/user_model_info.h

```diff
@@ -24,6 +25,7 @@ struct UserModelInfo final {
   SegmentList interest_segments;
   SegmentList latent_interest_segments;
   SegmentList purchase_intent_segments;
+  TextEmbeddingHtmlEventList embeddings_history;
```

Comment:
Can this not be named `text_embedding_html_events` to match the data structure name?

Reply:
That sounds great